### PR TITLE
some fixes

### PR DESCRIPTION
**Pull request by ma-efremoff**

### CODE REVIEW

File: packages/ui/src/ui/store/actions/acl.ts [hunk 1]
```diff
@@ -59,20 +57,10 @@ function getPathToCheckPermissions(
...
-                return `//sys/pool_trees/${poolTree}/${poolPath.join('/')}`;
-            } else {
-                const params = new URLSearchParams(window.location.search);
-                const path = params.get('path');
-                if (!path) {
-                    break;
-                }
-                return path;
-            }
+            const fullPath: string = await ytApiV3Id.get(YTApiId.schedulingPoolFullPath, {
```
Comment (SimbiozizV) [inline packages/ui/src/ui/store/actions/acl.ts:60]: error handling?

File: packages/ui/src/ui/common/thor/getNumber.spec.js [hunk 2]
```diff
@@ -65,5 +69,13 @@ describe('./thor/utils', () => {
                 expect(getNumber(node, '/@test_null', NaN)).toBe(NaN);
             });
         });
+
+        describe('yson', () => {
+            it('test', () => {
```
Comment (SimbiozizV) [inline packages/ui/src/ui/common/thor/getNumber.spec.js:74]: the name doesn't show the test goal

File: packages/ui/src/ui/store/actions/acl.ts (hunk 1 above)
Comment (ma-efremoff) [reply packages/ui/src/ui/store/actions/acl.ts:60]: It is by design, the code should throw error.

File: packages/ui/src/ui/common/thor/getNumber.spec.js (hunk 2 above)
Comment (SimbiozizV) [reply packages/ui/src/ui/common/thor/getNumber.spec.js:74]: number contert test?

**Review by SimbiozizV — APPROVED** [review]

**Pull request merged**